build/constants.yaml

```diff
@@ -8,6 +8,8 @@
     user-app-insights-key-env: APPINSIGHTS_INSTRUMENTATIONKEY
     user-app-insights-agent-extension-version: ApplicationInsightsAgent_EXTENSION_VERSION
     user-app-insights-connection-string-env: APPLICATIONINSIGHTS_CONNECTION_STRING
+    user-dotnet-startup-hooks: /DotNetCoreAgent/2.8.39/StartupHook/Microsoft.ApplicationInsights.StartupHook.dll
```

Comment:
`2.8.39` as a version number, we do want to generalize it so that easy for us to update next time or it's a fixed one?

Reply:
no we want to keep the version for now .. this is going to be an experimental use case .. so we will keep it